src/index.js

```diff
@@ -4,7 +4,13 @@ import insertStyle, { removeStyle, getStyle } from './insert-style';
 
 const cache = new Map();
 
-export default function withStyles(css) {
+export default function (css, values) {
+
+  if (Array.isArray(css)) {
+    // Forward string literals to csjs
+    css = csjs(css, values);
```

Comment:
Right, but this breaks a lint rule `no-param-reassign`. If you lint this branch it should complain about this line (if it doesn't, something is wrong with the linter), so I suggest to do something like:

```js
export default function (userCss, values) {
  let css = userCss;
  
  if (Array.isArray(css)) {
  ...
```